### ISSUE TITLE
accessilibity: WCAG 4.1.2: Ensure every form element has a label (#script-selector,.combobox-input[spellcheck="false"][type="text"])

### ISSUE THREAD

**Issue by pelikhan** [vscode-elements/elements#319]
WCAG 4.1.2: Ensure every form element has a label (#script-selector,.combobox-input[spellcheck="false"][type="text"])
Tags: Accessibility, WCAG 4.1.2, label

Issue: Ensure every form element has a label (label - https://accessibilityinsights.io/info-examples/web/label)

Target application: GenAIScript Script Runner - http://127.0.0.1:8003/?scriptid=add.test

Element path: #script-selector,.combobox-input[spellcheck="false"][type="text"]

Snippet: <input class="combobox-input" spellcheck="false" type="text">

How to fix: 
Fix any of the following:
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element has no placeholder attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Microsoft Edge version 133.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by pelikhan** [comment]
![Image](https://github.com/user-attachments/assets/8a399ebf-de57-410a-a11e-c37a0f57a6c7)
